src/schemas/json/haxelib.json

```diff
@@ -0,0 +1,93 @@
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"description": "A haxelib project",
+	"type": "object",
+	"properties": {
+		"name": { "$ref": "#/definitions/projectName" },
+		"url": {
+			"description": "Project's website",
+			"type": "string",
+			"format": "uri"
+		},
+		"description": {
+			"description": "Short description of the project",
+			"type": "string"
+		},
+		"license": {
+			"description": "Open source license under which the project is licensed",
+			"enum": ["GPL", "LGPL", "BSD", "Public", "MIT", "Apache"]
```

Comment:
This is correct, haxelib only allows a finite set of licenses that are defined in an enum too:

```haxe
@:enum abstract License(String) to String {
	var Gpl = 'GPL';
	var Lgpl = 'LGPL';
	var Mit = 'MIT';
	var Bsd = 'BSD';
	var Public = 'Public';
	var Apache = 'Apache';
}
```